src/AbstractAlgebra.jl

```diff
@@ -318,13 +318,15 @@ export is_trivial
 @alias is_trivial istrivial
 
 # alternative names for some functions from LinearAlgebra
+# we don't use the `@alias` macro here because we provide custom
+# docstrings for these aliases
```

Comment:
`@alias A B` does roughly three things:
- `const A = B` (which we want and I replicate below)
- if `B` has been exported, then it exports `B` -- but we already do that
- add a generic docstring to `A` just saying "A is an alias for B" -- but we don't want that, because we already have a *better* docstring for `A` in place (in particular, instead of "see B" it says "see LinearAlgebra.B", and actually describes the specific behavior we implement for our types